apps/ui/src/components/EditorVote.vue

```diff
@@ -0,0 +1,118 @@
+<script setup lang="ts">
+import Draggable from 'vuedraggable';
+import { BASIC_CHOICES, SUPPORTED_VOTING_TYPES } from '@/helpers/constants';
+import { Draft, VoteType } from '@/types';
+
+const proposal = defineModel<Draft>({ required: true });
+
+defineProps<{
+  error?: Record<number, string>;
+}>();
+
+const VOTE_TYPES = {
+  basic: {
+    label: 'Basic voting',
+    description: 'Single choice voting with three choices: For, Against or Abstain'
```

Comment:
```suggestion
    description: 'Single choice voting with three choices: For, Against or Abstain.'
```

I think because other options have `.` at the end.